README.md

```diff
@@ -53,9 +53,11 @@ __Note:__ You will need CocoaPods 1.0 beta4 or higher.
 A WKZombie instance equates to a web session, which can be created using the following line:
 
 ```ruby
-let browser = WKZombie(name: "Demo")
```

Comment:
I tried doing this at first – and it caused crashes, because browser was removed from memory before request finished.